src/gui/mzroll/settingsform.cpp

```diff
@@ -221,6 +221,11 @@ SettingsForm::SettingsForm(QSettings* s, MainWindow *w): QDialog(w) {
             &SettingsForm::loadModel);
     _updateModelPath();
 
+    connect(autosaveCheckBox,
```

Comment:
Should we connect _"use default parameters"_ in settings dialog to uncheck auto-save checkbox by default??

Reply:
Hm... no. I would want the user to make this decision and then it should stay that way until they chose to change it. "Use default parameters" is as I understand only for algorithm parameters.